composer.json

```diff
@@ -24,7 +24,7 @@
   "require": {
     "php": ">=7.2",
     "ext-soap": "*",
-    "scn/evalanche-soap-api-struct": "^1",
+    "scn/evalanche-soap-api-struct": "^1.1",
```

Comment:
I think we should leave that on version 1, features of the Struct have no effect here.

Reply:
Nope, this change *requires* 1.1 to be installed. If someone blocks struct >=1.1 (dont as why, just theoretical), the new version of the connector must not be installed because it relys on the structs.